src/extensions/filtercontrol/bootstrap-table-filtercontrol.js

```diff
@@ -113,12 +114,43 @@
                         break;
                     case 'select':
                         html.push(sprintf('<select class="%s form-control" style="width: 100%; visibility: %s"></select>',
-                            column.field, isVisible))
+                            column.field, isVisible));
                         break;
                 }
             }
 
             that.$header.find(sprintf('.th-inner:contains("%s")', column.title)).next().append(html.join(''));
+            if(column.filterData !== undefined && column.filterData.toLowerCase() !== 'column'){
+                var filterDataType = column.filterData.substring(0, 3);
```

Comment:
And if we have a relative url?
For example let's say my website is located to www.domain.com, and location to data that i want to load is located to www.domain.com/data/bootstrap-table.json, then i can set data to select in two ways:
1. url:http://www.domain.com/data/bootstrap-table.json - absolute path
2. url:/data/bootstrap-table.json - relative to current domain
What do you think?